### PR TITLE
add wavefront-proxy

### PR DESCRIPTION
**Pull request by ajayk**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [x] REQUIRED - The package is added to `packages.txt`

### CODE REVIEW

File: wavefront-proxy.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  - runs: |
+      export LANG=en_US.UTF-8
+      export JAVA_HOME=/usr/lib/jvm/openjdk
+      mvn -f proxy --batch-mode clean package -DskipTests -DskipFormatCode
+      mkdir -p ${{targets.destdir}}/opt/wavefront/wavefront-proxy
+      mv /home/build/proxy/target/proxy-${{package.version}}-spring-boot.jar ${{targets.destdir}}/opt/wavefront/wavefront-proxy/wavefront-proxy.jar
+
+subpackages:
+  - name: ${{package.name}}-config
```
Comment (kaniini) [inline wavefront-proxy.yaml:37]: Why not `name: wavefront-proxy-config`?

**Review by imjasonh — DISMISSED** [review]

File: wavefront-proxy.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    with:
+      repository: https://github.com/wavefronthq/wavefront-proxy
+      tag: proxy-${{package.version}}
+      expected-commit: 7170d26b802e5f4b2201c3a38c2d11e6e3ad719a
+
+  - runs: |
+      export LANG=en_US.UTF-8
+      export JAVA_HOME=/usr/lib/jvm/openjdk
+      mvn -f proxy --batch-mode clean package -DskipTests -DskipFormatCode
+      mkdir -p ${{targets.destdir}}/opt/wavefront/wavefront-proxy
```
Comment (kaniini) [inline wavefront-proxy.yaml:33]: Packages should not go into `/opt`.

**Review by kaniini — CHANGES_REQUESTED** [review]

File: wavefront-proxy.yaml (hunk 2 above)
Comment (ajayk) [reply wavefront-proxy.yaml:33]: the [run.sh](https://github.com/wavefrontHQ/wavefront-proxy/blob/master/docker/run.sh#L62) has a reference /opt , so we would need to override/re-write the run.sh when we do an image with a new reference ?

The only other package reference I found was from conda where the package was going to /opt

Comment (dlorenc) [reply wavefront-proxy.yaml:33]: We could do it in a compat subpackage if they hard code it.

Comment (ajayk) [reply wavefront-proxy.yaml]: I moved the packages to `/usr/share/java/wavefront/wavefront-proxy` add a replace in the compat subpackage

**Review by rawlingsj — DISMISSED** [review]

**Review by rawlingsj — DISMISSED** [review]

**Review by imjasonh — APPROVED** [review]

**Review by kaniini — APPROVED** [review]

**Pull request merged**